src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int COLOR_COUNT = 5;
     public String getRandomColor() {
-        return null;
+        return Color.values()[COLOR_COUNT].name();
```

Comment:
you should use dynamic number instead of COLOR_COUNT which is always 5, you need to generate the number first randomly